core/misc/backdrop.js

```diff
@@ -446,6 +446,60 @@ Backdrop.ajaxError = function (xmlhttp, uri, customMessage) {
   return message;
 };
 
+/**
+ * Check to see when webfont has loaded and adjust the tabs display
+ * @param fontName
+ *   Font name as shown in CSS
+ * @param callback
+ *   Function to run once font has loaded
+ */
+if (typeof Backdrop.fontsLoaded === 'undefined') {
+  Backdrop.fontsLoaded = {};
+}
+Backdrop.isFontLoaded = function(fontName, callback) {
+  if (typeof fontName === 'undefined') {
+    return;
+  }
+
+  if (typeof Backdrop.fontsLoaded[fontName] === 'undefined') {
+    Backdrop.fontsLoaded[fontName] = false;
+  }
+  else if (Backdrop.fontsLoaded[fontName]) {
+    // Fonts loaded, run the callback and don't run anything else.
+    callback();
+    return;
+  }
+
+  var $body = $('body');
+  var checkFontCounter = 0;
+  // Append an invisible element that will be monospace font or our desired
+  // font. We're using a repeating i because the characters width will
+  // drastically change when it's monospace vs. proportional font.
+  var $checkFontElement = $('<span id="check-font" styles="font-family: ' + fontName + ', monospace; position: absolute; z-index: -100; visibility: hidden;">iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii</span>');
```

Comment:
Happy to see this moved out of the Seven theme and into a JS file where it won't trouble themers so directly.